BUILD.md

```diff
@@ -262,12 +262,72 @@ It patches their CMake to correctly import its dependencies.
    generator. Pass `--help` to see the rest of the command line options.
 
 
+## Coverage report
+
+Coverage report is intended for the developers using compilers GCC
```

Comment:
```suggestion
The coverage report is intended for the developers using compilers GCC
```

Reply:
The are many other instances in this section where the definite article is omitted. I think it'd be good to have a technical writer edit the whole section for grammar and style.